frontend/src/components/TaskCreator/DatePicker.tsx

```diff
@@ -132,15 +132,16 @@ export default function DatePicker(props: Props): ReactElement {
       </>
     );
     return (
-      <span
-        role="presentation"
+      <button
+        // role="presentation"
```

Comment:
Should remove commented code before merging to master, especially if it should be deleted